apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsDetail.java

```diff
@@ -95,15 +96,70 @@
   @Column(name = "`idrKPosZip`", length = 15)
   private String idrKPosZip;
 
+  @Column(name = "`idrTos`", length = 1)
+  private String idrTos;
+
+  @Column(name = "`idrTwoDigitPos`", length = 2)
+  private String idrTwoDigitPos;
+
+  @Column(name = "`idrDtlRendType`", length = 2)
+  private String idrDtlRendType;
+
+  @Column(name = "`idrDtlRendSpec`", length = 2)
+  private String idrDtlRendSpec;
+
+  @Column(name = "`idrDtlRendNpi`", length = 10)
+  private String idrDtlRendNpi;
+
+  @Column(name = "`idrDtlRendProv`", length = 10)
+  private String idrDtlRendProv;
+
+  @Column(name = "`idrKDtlFacProvNpi`", length = 10)
```

Comment:
🍰  Field name comments might help on column names like this one too!

Reply:
I grabbed those comments from the RDA API proto files. Not all of the fields have comments though.